test_llava.py

```diff
@@ -0,0 +1,21 @@
+import requests
```

Comment:
Can we move `test_llava.py` and `test_llava_quantized.py` to `scripts/models`? Could we remove the `test_` prefix from the file name?

Reply:
Maybe remove one of them since they are pretty similar.

Reply:
oh sorry I meant to remove these actually